src/sentry/audit_log/register.py

```diff
@@ -565,25 +565,49 @@
 default_manager.add(
     AuditLogEvent(
         event_id=210,
-        name="WORKFLOW_ENGINE_DETECTOR_ADD",
-        api_name="workflow_engine_detector.add",
-        template="added workflow engine detector {name}",
+        name="DETECTOR_ADD",
+        api_name="detector.add",
+        template="added detector {name}",
```

Comment:
should we say who added/modified/delete the detector/workflow?

Reply:
I believe this is already covered when you call `create_audit_entry`, it gets the user from the request https://github.com/getsentry/sentry/blob/master/src/sentry/utils/audit.py#L42-L44

Reply:
ah i c

Reply:
Hmm yeah I could make a new entry type for connecting a detector to a workflow, will we have an API for that?

Reply:
unclear! we can wait on that